src/PRASBase/assets.jl

```diff
@@ -280,3 +398,44 @@ Base.:(==)(x::T, y::T) where {T <: Lines} =
     x.backward_capacity == y.backward_capacity &&
     x.λ == y.λ &&
     x.μ == y.μ
+
+Base.getindex(lines::LINE, idxs::AbstractVector{Int}) where {LINE <: Lines} =
```

Comment:
Same as above, could `LINE` just be `L`?  I know that doesn't work in `vcat` below but hopefully we can eliminate the need for the type variable there anyways based on my first comment.

Reply:
I posted this comment too soon - read the next comment first if this seems confusing